src/index.js

```diff
@@ -28,7 +32,7 @@ const randomUser = () => {
 }
 
 const onLoad = () => {
-  Promise.all([getRooms(), getBookings(), getSingleUser(randomUser())])
+  Promise.all([getRooms(), getBookings(), getSingleUser(5)])
```

Comment:
It's interesting that you used functions here for each independent fetch request. That probably makes your getSingleUser function more dynamic. I like it! It might be helpful to separate all your fetch functions into in an API file that can hold all  your requests, POST and GET, etc.